EpiAware/src/latent-processes.jl

```diff
@@ -1,14 +1,14 @@
 function default_rw_priors()
     return (
-        var_RW_dist = truncated(Normal(0.0, 0.05), 0.0, Inf),
-        init_rw_value_dist = Normal()
+        var_RW_prior = truncated(Normal(0.0, 0.05), 0.0, Inf),
```

Comment:
note we have a lot of repetition in our name. We have a function that is called default_rw_priors and then everything in it is called rw as are all the args in the random walk function.

Reply:
I was just thinking this too. I'll spin out your comment to an issue.